hack/gen-versions/components.go

```diff
@@ -32,11 +32,11 @@ var defaultImages = map[string]string{
 	"flannel":                    "coreos/flannel",
 	"flexvol":                    "calico/pod2daemon-flexvol",
 	"typha":                      "calico/typha",
-	"eck-elasticsearch":          "tigera/elasticsearch",
-	"eck-elasticsearch-operator": "tigera/eck-operator",
-	"eck-kibana":                 "tigera/kibana",
-	"coreos-prometheus":          "tigera/prometheus",
-	"coreos-alertmanager":        "tigera/alertmanager",
+	"eck-elasticsearch":          "unused/image",
```

Comment:
Why are we changing this to unused? We still install these images, defaults are still nice to have...

Reply:
We don't use these images. Take a look at the entries in the versions yaml https://github.com/tigera/operator/blob/master/config/enterprise_versions.yml, we don't include an image for the entries changed to `unused/image` because we only use the version field for those entries. You'll find a similar entry without `eck` or `coreos` that we do use that has an image field.

Reply:
Right right, thanks for the explanation.